website/content/docs/audit/index.mdx

```diff
@@ -56,6 +56,8 @@ In the command above, we passed the "file_path" parameter to specify the path
 where the audit log will be written to. Each audit device has its own
 set of parameters. See the documentation to the left for more details.
 
+~> Note: Audit device configuration is replicated to all nodes within a cluster by default. Before enabling an audit device, ensure that all nodes within the cluster will be able to successfully log to the audit device to avoid Vault being blocked from serving requests. An audit device can be enabled as a local device with the [`local`](api/system/audit#local) parameter. 
```

Comment:
The first part of this paragraph seems to refer to perf standby nodes ("nodes within a cluster").  The last sentence is about audit devices across performance secondaries.  This might be confusing, maybe break them up?

Reply:
Oh, I might misunderstand `local`.
"Specifies if the audit device is a local only"

Is that local to the node or local to the cluster?

Reply:
Local to the cluster.

Reply:
So given my new understanding of `local`... By default, does an audit enable replicate to a secondary?

Reply:
Yes.

Reply:
Cleaning up backlogs and merging doc-related PRs that have been approved and ready to be merged.